data/tlg0011/tlg003/tlg0011.tlg003.1st1K-grc1.xml

```diff
@@ -7,7 +7,7 @@
             <title type="work" n="Aj">Ajax</title>
```

Comment:
There are a lot of little differences here to what we usually do. How much does this matter?